lib/cli/src/Cardano/CLI.hs

```diff
@@ -1217,57 +1228,65 @@ instance FromText TxId where
 
 -- | Controls how much information to include in log output.
 data Verbosity
-    = Default
-        -- ^ The default level of verbosity.
-    | Quiet
+    = Quiet
         -- ^ Include less information in the log output.
     | Verbose
         -- ^ Include more information in the log output.
     deriving (Eq, Show)
 
--- | Convert a given 'Verbosity' level into a list of command line arguments
---   that can be passed through to a sub-process.
-verbosityToArgs :: Verbosity -> [String]
-verbosityToArgs = \case
-    Default -> []
-    Quiet   -> ["--quiet"]
-    Verbose -> ["--verbose"]
-
 -- | Map a given 'Verbosity' level onto a 'Severity' level.
 verbosityToMinSeverity :: Verbosity -> Severity
 verbosityToMinSeverity = \case
-    Default -> Info
-    Quiet   -> Error
     Verbose -> Debug
+    Quiet -> Error
+
+allTracers :: [Text]
+allTracers = map ("cardano-wallet" <>)
```

Comment:
Wouldn't it be better from a user experience that the "cardano-wallet" is added on top of the user inputs? As far as I understand it, if I want to make the api more verbose, I'll do: `cardano-wallet ... --verbose cardano-wallet.api` whereas, it sounds slightly better to only have to do `cardano-wallet ... --verbose api`, the `cardano-wallet` is already implied by the surrounding context. No ?